code.js

```diff
@@ -1,3 +1,7 @@
 function binarySearch(list, element) {
+    for(var i = 0; i < list.length; i++){
```

Comment:
This is not Binary Search. You need to find the mid and compare the element with mid and then divide the array into half according to the comparison with mid element.